### PR TITLE
Add style class without height for Ticket page

### PR DESCRIPTION
**Pull request by mileTadic1994**
This pull request resolves issue #99 
Here is video how it looks with more ticket types

[2019_06_27_13_38_15.mp4.tar.gz](https://github.com/big-neon/bn-mobile-react/files/3334632/2019_06_27_13_38_15.mp4.tar.gz)

And here is how it looks with less
Screenshot_20190627-140837_Expo

### CODE REVIEW

File: src/Events/tickets.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default class GetTickets extends Component {
   }
 
   onPromoApply = async (code = '') => {
-    if (code === '') {
+    if (!code) {
```
Comment (Krakaw) [inline src/Events/tickets.js]: What happens if they enter the code `0` will not checking the type not be a problem?

Comment (mileTadic1994) [reply src/Events/tickets.js]: Yes, you are right. But I thought there is no case that code will ever be one of null values.

Comment (Krakaw) [reply src/Events/tickets.js]: What about doing something along the lines of
```javascript
if (String(code).trim() === '')
```

Comment (mileTadic1994) [reply src/Events/tickets.js]: Its okay if you want to ignore whitespaces

**Comment by mileTadic1994** [comment]
> See the attached video.

Fixed

**Pull request merged**